### ISSUE TITLE
Cross-validation sets

### ISSUE THREAD

**Issue by javierperezcastan** [pycaret/pycaret#875]
Hi, 

I would like to know how apply the cross-validation during the model evaluation. After applying the setup function of Pycaret I have a training and test set. When I create a model, for instance a decission tree, a cross-validation is performed to obtain the average value of the metrics. Is the CV applied to the whole dataset (training + test) or just in the trainnig set?

**Comment by Yard1** [comment]
CV is only applied to the training set. No part of the test set is used for any model fitting whatsoever.

**Comment by pycaret** [comment]
@javierperezcastan As @Yard1 confirmed the CV is only applied on the train set.

Please feel free to open another issue, if you have more questions.
